File: 1.FDP/2° Competencia/C++/Seleccion/Ejercicio1.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Calcular el cambio de una compra.\n";
	double Total, Pago, Cambio;
	cout << "Ingresar el total:$";
	cin >> Total;
	if (Total > 0){
		cout << "Ingresar el pago:$";
		cin >> Pago;
		if (Pago >= Total ) {
			Cambio= Pago-Total;
			cout << "Cambio = $" << Cambio << "\n";	
		} else{ 
		cout << "Error en el pago.\n";
		
		}
	} else {
		cout << "Error en el total.\n";
		
	}
	
	return 0;
}

File: 1.FDP/2° Competencia/Java/Seleccion/Ejercicio1.java
import java.util.Scanner;	

public class Ejercicio1 {
	public static void main(String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular el cambio de una compra.\n");
		double Total, Pago, Cambio;
		System.out.println("Ingresar el total de compra:");
		Total= scan.nextDouble();
		if (Total > 0) {
			System.out.println("Ingresar la cantidad de pago:");
			Pago= scan.nextDouble();
			if (Pago >= Total) {
				Cambio= Pago-Total;
				System.out.println("Cambio= " + Cambio);
			}else{
				System.out.println("Error en el total");
			}
		}else{
			System.out.println("Error en el pago.\n");
		}
		//
	}
}